compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -816,19 +816,19 @@ class Definitions {
   // A list of annotations that are commonly used to indicate that a field/method argument or return
   // type is not null. These annotations are used by the nullification logic in JavaNullInterop to
   // improve the precision of type nullification.
-  @tu lazy val NotNullAnnots: List[ClassSymbol] =
-    ("javax.annotation.Nonnull" ::
-      "edu.umd.cs.findbugs.annotations.NonNull" ::
-      "androidx.annotation.NonNull" ::
-      "android.support.annotation.NonNull" ::
-      "android.annotation.NonNull" ::
-      "com.android.annotations.NonNull" ::
-      "org.eclipse.jdt.annotation.NonNull" ::
-      "org.checkerframework.checker.nullness.qual.NonNull" ::
-      "org.checkerframework.checker.nullness.compatqual.NonNullDecl" ::
-      "org.jetbrains.annotations.NotNull" ::
-      "lombok.NonNull" ::
-      "io.reactivex.annotations.NonNull" :: Nil).map(ctx.requiredClass(_))
+  @tu lazy val NotNullAnnots: List[ClassSymbol] = ctx.getClassesIfDefined(
```

Comment:
Could you add the following sentence to the comment

We don't require that any of these annotations be present in the class path, but we want to
create Symbols for the ones that _are_ present, so they can be checked during nullification.